csm_test_utils/rds/rds_backup.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+import logging
+import sys
+import time
+import requests
+import yaml
+import json
+
+from influx_line_protocol import Metric, MetricCollection
+from ocomone import setup_logger
+from requests import Response, HTTPError
+from datetime import datetime
+from ..common import Client, base_parser, sub_parsers
+
+
+BASE_URL = "https://rds.eu-de.otc.t-systems.com/v3"
+RDS_BACKUP = "rds_backup_monitor"
+CSM_EXCEPTION = "csm_exception"
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+
+def get_auth_token(cloud_config, cloud_name) -> str:
+    """Get auth token using data from clouds.yaml file. Token and project_id are returned as a string"""
+    with open(cloud_config) as clouds_yaml:
+        data = yaml.safe_load(clouds_yaml)
+    request_headers = {'Content-Type': 'application/json;charset=utf8'}
```

Comment:
make variable for 'Content-Type': 'application/json;charset=utf8'

Reply:
Done